tests/conftest.py

```
@@ -44,12 +58,15 @@ def flask_app_url():
 
     yield BASE_URL
 
-    if os.name == 'nt':
-        subprocess.run(["taskkill", "/F", "/PID", str(process.pid)], capture_output=True)
+    if os.name == "nt":
+        subprocess.run(
```

Ruff (0.14.4)

`subprocess` call: check for execution of untrusted input

(S603)